apps/server/src/modules/room/api/test/room-change-role.api.spec.ts

```diff
@@ -186,5 +190,37 @@ describe('Room Controller (API)', () => {
 				expect(response.status).toBe(HttpStatus.BAD_REQUEST);
 			});
 		});
+
+		describe('when the user is a school admin', () => {
+			const setupAdminLogin = async () => {
+				const { room, school, targetUser, externalTeacherUser } = await setupRoomWithMembers();
+				const { adminAccount, adminUser } = UserAndAccountTestFactory.buildAdmin({ school });
+				await em.persistAndFlush([adminAccount, adminUser]);
+				const loggedInClient = await testApiClient.login(adminAccount);
+				return { loggedInClient, room, targetUser, externalTeacherUser };
+			};
+
+			it('should not allow to pick role roomowner (=> different endpoint)', async () => {
```

Comment:
- remove hint `(=> different endpoint)`?
- here its named "pick role", below we call it "passing ownership", maybe unify?

Reply:
fixed